wgsl/index.bs

```diff
@@ -650,11 +650,11 @@ type, function decorations and storage class.
     <tr><td>front_facing<td>bool<td>Fragment Input
     <tr><td>frag_coord<td>vec4<f32><td>Fragment Input
     <tr><td>frag_depth<td>f32<td>Fragment Output
-    <tr><td>num_workgroups<td>vec3<i32><td>Compute Input
-    <tr><td>workgroup_size<td>vec3<i32><td>Compute Input
-    <tr><td>local_invocation_id<td>vec3<i32><td>Compute Input
-    <tr><td>global_invocation_id<td>vec3<i32><td>Compute Input
-    <tr><td>local_invocation_idx<td>i32<td>Compute Input
+    <tr><td>num_workgroups<td>vec3<u32><td>Compute Input
```

Comment:
Can you fix these so they render correctly with bikeshed? The <u32> will get picked up as a tag and stripped out at the moment.

Reply:
I don't understand. Just loaded the HTML and it shows the table just fine. What are we missing here exactly?

Reply:
It shows 

num_workgroups | vec3 | Compute Input
-- | -- | --

Instead of

num_workgroups | vec3\<u32\> | Compute Input
-- | -- | --

A backslash solves in Markdown, not sure about BS